### PR TITLE
Added some test cases to first function in utils.py

### PR DESCRIPTION
**Pull request by sagar-sehgal**
Added more test cases to first function

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -35,9 +35,14 @@ def test_first():
     assert first('word') == 'w'
     assert first('') is None
     assert first('', 'empty') == 'empty'
+    assert first([1,2,3,4,5]) == 1
```
Comment (MrDupin) [inline tests/test_utils.py]: To conform with PEP8, you should add a space after each comma.

File: tests/test_utils.py [hunk 2]
```diff
@@ -35,9 +35,14 @@ def test_first():
...
+    assert first([]) == None
     assert first(range(10)) == 0
     assert first(x for x in range(10) if x > 3) == 4
     assert first(x for x in range(10) if x > 100) is None
+    assert first((1,2,3)) == 1
```
Comment (MrDupin) [inline tests/test_utils.py]: Here as well.

File: tests/test_utils.py [hunk 3]
```diff
@@ -35,9 +35,14 @@ def test_first():
...
+    assert first([(1,2),(1,3),(1,4)]) == (1,2)
```
Comment (MrDupin) [inline tests/test_utils.py]: And here.

File: tests/test_utils.py [hunk 4]
```diff
@@ -35,9 +35,14 @@ def test_first():
...
+    assert first({1:"one",2:"two",3:"three"}) == 1
```
Comment (MrDupin) [inline tests/test_utils.py]: And here.

File: tests/test_utils.py (hunk 1 above)
Comment (sagar-sehgal) [reply tests/test_utils.py]: Ok @MrDupin , I'll do that .

**Pull request merged**